pkg/assets/assets_generate.go

```diff
@@ -23,8 +23,12 @@ import (
 
 func main() {
 	dirs := map[string]string{
-		"/lokomotive-kubernetes": "../../assets/lokomotive-kubernetes",
-		"/components":            "../../assets/components",
+		assets.TerraformModulesSource: "../../assets/lokomotive-kubernetes",
```

Comment:
We could also move this assets to something like `assets/terraform-modules` ?

Reply:
Done.